test/test_type_hints.py

```diff
@@ -170,6 +170,30 @@ def test_doc_examples(self):
             except subprocess.CalledProcessError as e:
                 raise AssertionError("mypy failed.  Look above this error for mypy's output.")
 
+    @unittest.skipIf(sys.version_info[0] == 2, "no type hints for Python 2")
+    @unittest.skipIf(not HAVE_MYPY, "need mypy")
+    def test_type_hint_examples(self):
+        """
+        Runs mypy over all the test examples present in
+        `type_hint_tests` directory.
+        """
+    test_path = os.path.dirname(os.path.realpath(__file__))
+    examples_folder = os.path.join(test_path, "type_hint_tests")
+    examples = os.listdir(examples_folder)
+    for example in examples:
+        example_path = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                                    "type_hint_tests", example)
+        try: 
+            example_path = os.path.join(examples_folder, example)
+            subprocess.run([ 
+                sys.executable, 
+                '-mmypy', 
+                '--follow-imports', 'silent', 
+                '--check-untyped-defs', 
+                example_path],  
+                check=True) 
+        except subprocess.CalledProcessError as e: 
+            raise AssertionError("mypy failed for example {}.  Look above this error for mypy's output.".format(example))
 
 if __name__ == '__main__':
-    run_tests()
+    run_tests()
```

Comment:
Flake8 is complaining that there is no newline here. We should add a newline (hit enter at the end of run_tests())

Reply:
I've done as you said.